indexBasecodeANSWER1c.html

```diff
@@ -0,0 +1,157 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="utf-8" />
+		<title>Flights to Mars</title>

```

Comment:
This is an information, we could have used a linter here that will align the tags in the right position after self closing tags like the <meta ../> above.